frontend/styles/global.styles.ts

```diff
@@ -0,0 +1,39 @@
+import { createGlobalStyle } from "styled-components";
+
+export const GlobalStyles = createGlobalStyle`
+    @import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600;700;800;900&display=swap');    
+    body{
+        background: ${({ theme }) => theme.body};
+        color: ${({ theme }) => theme.text};
+        font-family: 'Inter', sans-serif;
+    }
+
+    h1{
```

Comment:
Will this end up really big? Can we maybe break down the global functional styles into little sub directories? (Idk best practice for this stuff so I might be saying something really stupid lol)

Reply:
Yeah, most likely

Reply:
.. That won't be too big. They will act as our root css or css normalizer. We wont have to change it later. We can export css in styled components so this wont be a problem. If in case it gets too large we can break it. But dont see the point now as I dont think it will get large.